src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -274,16 +274,18 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                     var isStyleEl  = domUtils.isStyleElement(el);
                     var isScriptEl = domUtils.isScriptElement(el);
 
-                    if (value) {
+                    var processedValue = String(value);
```

Comment:
We need to return origin value.
\> script.innerHTML = {a: 1}; // return object {a: 1}
\> script.innerHTML === '[object Object]' // true
[see test](https://github.com/DevExpress/testcafe-hammerhead/pull/1092/files/a80cab1cf06b72e4f67393f84a2eccf0ccf985e3#diff-bbfd287e468ceb547f28a177a62cb879R280)